### PR TITLE
Finch 0.10.0

### PR DESCRIPTION
**Pull request by mfirry**
[finch](https://github.com/finagle/finch]) bumped to 0.10.0.

### CODE REVIEW

File: frameworks/Scala/finch/src/main/scala/WebServer.scala [hunk 1]
```diff
@@ -14,25 +11,25 @@ import io.circe._
 import io.circe.generic.auto._
 import io.circe.jawn._
 import io.circe.syntax._
-import io.finch.response.EncodeResponse
 import io.circe.{Decoder, Encoder, Json}
 
 object WebServer extends App {
 
+  import io.finch.circe._
   val json = get("json") {
```
Comment (vkostyukov) [inline frameworks/Scala/finch/src/main/scala/WebServer.scala]: Can we put type here `Endpoint[io.circe.Json]` and return `Json.obj("message" -> Json.string("Hello, World!"))` directly?

File: frameworks/Scala/finch/src/main/scala/WebServer.scala [hunk 2]
```diff
@@ -14,25 +11,25 @@ import io.circe._
...
 
-  val plaintext = get("plaintext") {
-    "Hello, World!"
+  val plaintext: Endpoint[String] = get("plaintext") {
+    Ok("Hello, World!")
+      .withContentType(Some("text/plain"))
   }
 
-  val api: Service[Request, Response] = (json :+: plaintext).toService
+  val api = json :+: plaintext
```
Comment (vkostyukov) [inline frameworks/Scala/finch/src/main/scala/WebServer.scala]: Let' inline this into `serve` call.

**Comment by vkostyukov** [comment]
Thanks @mfirry for taking care of this!

File: frameworks/Scala/finch/src/main/scala/WebServer.scala (hunk 2 above)
Comment (mfirry) [inline frameworks/Scala/finch/src/main/scala/WebServer.scala]: You mean `Http.serve(":9000", (json :+: plaintext).toService)`?

File: frameworks/Scala/finch/src/main/scala/WebServer.scala (hunk 1 above)
Comment (mfirry) [inline frameworks/Scala/finch/src/main/scala/WebServer.scala]: Consider it done.

File: frameworks/Scala/finch/src/main/scala/WebServer.scala (hunk 2 above)
Comment (vkostyukov) [inline frameworks/Scala/finch/src/main/scala/WebServer.scala]: Yup.

**Comment by mfirry** [comment]
Ideally one should disable gzip and add `Server` and `Date` headers.

**Comment by vkostyukov** [comment]
You can disable compression by `Http.server.withCompressionLevel(0)`. I should probably do the same for a Finagle PR I opened #1941. Also you can use that as an inspiration for `Date` and `Server` codecs.

**Comment by vkostyukov** [comment]
:+1: Thanks @mfirry! Looks great!

File: frameworks/Scala/finch/src/main/scala/WebServer.scala [hunk 3]
```diff
@@ -14,25 +13,27 @@ import io.circe._
...
+      .withHeader("Date" -> dateFormat.format(new Date()))
+      .withContentType(Some("text/plain"))
   }
 
-  val api: Service[Request, Response] = (json :+: plaintext).toService
-
-  Await.ready(
-    Httpx.serve(":9000", api)
+  Await.ready(Http.server
+    .withCompressionLevel(0)
```
Comment (vkostyukov) [inline frameworks/Scala/finch/src/main/scala/WebServer.scala]: Sorry to bother again, but can you also disable stats receiver and tracer on a server instance? (in a same way as per #1941).

```scala
.withStatsReceiver(NullStatsReceiver)
.withTracer(NullTracer)
``

**Comment by ssmith-techempower** [comment]
Looks good to me, merging in.

**Pull request merged**